### DISCUSSION TITLE
KPN IPTV Error 563

### DISCUSSION THREAD

**Discussion by leunampord** [fabianishere/udm-iptv#264] [Q&A]
Hi! 

After I managed to get the KPN VoIP working on the UDM pro i am now trying to get the IPTV working. 
I have set up an iptv VLAN (6) and connected it to the IPTV decoder from KPN/Xs4all and installed the udm-iptv program as explained here. IGMP snooping is enabled on the network. 
But the box keeps giving an error 563. "opstarten niet mogelijk" (boot not possible). The error means: "the tv receiver is getting the wrong IP address to reach the TV network".
I have tried adjusting the configuration from  KPN to Xs4all, but nothing changes it seems. 

What did I do wrong?

=== Configuration ===
WAN Interface: eth8
WAN VLAN: 4 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 213.75.0.0/16  217.166.0.0/16
LAN Interfaces: br0
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: true
=== IP Link and Route ===
58: iptv@eth8: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    inet 10.30.60.129/22 brd 10.30.63.255 scope global iptv
       valid_lft forever preferred_lft forever
10.30.60.0/22 proto kernel scope link src 10.30.60.129
213.75.112.0/21 via 10.30.60.1 metric 258
=== Service Logs ===
Jul 03 17:30:50 UDM-Pro udm-iptvd[366850]: The IGMP message was local multicast. Ignoring.
Jul 03 17:30:51 UDM-Pro udm-iptvd[366850]: RECV V2 member report   from 192.168.50.1    to 233.89.188.1
Jul 03 17:30:51 UDM-Pro udm-iptvd[366850]: The IGMP message was from myself. Ignoring.
Jul 03 17:30:51 UDM-Pro udm-iptvd[366850]: RECV V2 member report   from 192.168.50.1    to 224.0.0.22
Jul 03 17:30:51 UDM-Pro udm-iptvd[366850]: The IGMP message was from myself. Ignoring.
Jul 03 17:30:54 UDM-Pro udm-iptvd[366850]: RECV V2 member report   from 192.168.50.1    to 224.0.0.2
Jul 03 17:30:54 UDM-Pro udm-iptvd[366850]: The IGMP message was from myself. Ignoring.
Jul 03 17:30:57 UDM-Pro udm-iptvd[366850]: RECV Membership query   from 192.168.50.1    to 224.0.0.1
Jul 03 17:30:57 UDM-Pro udm-iptvd[366850]: RECV V2 member report   from 192.168.50.1    to 224.0.0.22
Jul 03 17:30:57 UDM-Pro udm-iptvd[366850]: The IGMP message was from myself. Ignoring.

**Comment by fabianishere** [comment]
If your STB is connected via VLAN 6, you need to select VLAN 6 (br6) as LAN interface in the config.

**Reply by leunampord** [reply]
Ah yes i see it says BR0, but i did select VLAN 6 when asked which LAN interface to use during udm-iptv configure.

**Reply by fabianishere** [reply]
You have to select it using spacebar, not enter as that will just proceed to the next screen.

**Reply by leunampord** [reply]
I didn't know how stupid i was, im sorry. Hopefully is this problem less stupid: now it works, kind off, most of the time the screen is black and when the television finally comes on, it stutters and doesn't run (smoothly).

**Reply by marcoheijkoop** [reply]
Did you configure the port that's connected to the box to deliver vlan 4&6 and nothing else. If have it like this https://dawnbringer.net/blog/1086/ubiquiti-usg-kpn-experia-iptv But the Experia box is gone now. Fully running on UDM Pro without problems

**Reply by leunampord** [reply]
I didn’t follow that guide because I always had internet trouble with the fritz box before my udm pro. So i now have a vdsl modem in bridge mode, then the udm-pro and after that the fritzbox, just for the VoIP. 
The tv boxes are connected to the udm (via a switch). 
the tv now does boot, and then it plays the tv feed after 10 seconds of black screen, for about half a minute, until the screen freezes or the image starts to break up. 
also during the Tv is on, the rest of the internet falls away. :( 

Something to do with firewall or network distribution? 
Thank you all for the help!
